### DISCUSSION TITLE
Atomatically mark as read

### DISCUSSION THREAD

**Discussion by david-corkill** [martinrotter/rssguard#877] [Ideas]
A feature I use a lot in other readers is the ability to enable 'automatically mark all feed articles as read'.. This may be something like 'after x seconds', or more importantly, 'when navigating away from the feed'

The use case is when I am just scanning the headlines to see if I want to read something. Once I have scanned the titles, I move to the next feed and want to save clicks marking all as read.

I have searched through the doco and couldn't find anything. For your consideration

**Comment by c-prompt** [comment]
+1. QuiteRSS hasn't been updated in so long and is constantly crashing so checked out RSS Guard. However, QuiteRSS has what I believe @david-corkill references: when a feed is highlighted with multiple unread articles (for which I'm not interested), clicking on the next feed automatically marks all articles in the previously highlighted feed as read. As I understand how it works in RSS Guard, you must perform two actions: 1) for the feed, "Mark selected item read" and, then; 2) click on the next feed. It would be more efficient for RSS Guard to have functionality that encompasses both actions. Ideally, just enable with a Settings checkbox that moving to a different feed marks all items in the previous feed read. Thanks.
